packages/manager/modules/pci/src/projects/project/storages/object-storage/users/translations/Messages_fr_FR.json

```
@@ -1,10 +1,12 @@
 {
-  "pci_projects_project_storages_containers_userlist_title": "Ajouter des utilisateurs",
-  "pci_projects_project_storages_containers_userlist_username": "Nom d'utilisateu",
-  "pci_projects_project_storages_containers_userlist_description": "Description",
-  "pci_projects_project_storages_containers_userlist_import_json": "Importer un fichier Json",
-  "pci_projects_project_storages_containers_userlist_download_json": "Télécharger le fichier json",
-  "pci_projects_project_storages_containers_userlist_user_management": "Gestion des utilisateurs",
-  "pci_projects_project_storages_containers_userlist_user_description": "Afin d'ajouter de nouveaux utilisateurs vous devez les créer dans la sectionn «Users & Roles»",
-  "pci_projects_project_storages_containers_userlist_add_user": "Ajouter des utilisateurs"
+  "pci_projects_project_storages_containers_users_title": "Ajouter des utilisateurs",
+  "pci_projects_project_storages_containers_users_username": "Nom d'utilisateu",
+  "pci_projects_project_storages_containers_users_description": "Description",
+  "pci_projects_project_storages_containers_users_import_json": "Importer un fichier Json",
```

```suggestion
  "pci_projects_project_storages_containers_users_import_json": "Importer un fichier JSON",
```
please be consistent across all content for `JSON`